laravel/app/Models/Inventory.php

```diff
@@ -57,15 +57,19 @@ public function scopeTemporaryIndex($query, $item_code, $delivery_user_id)
 		return $query;
 	}
 
-	public function scopeTemporaryShip($query, $ship_date, $change, $change_id, $item_ids)
+	public function scopeTemporaryShip($query/*, $item_ids, $ship_date, $change, $change_id*/)
```

Comment:
引数を一部コメントアウトしたいときは、わかりやすさ重視でこう書きましょう
```suggestion
	//public function scopeTemporaryShip($query, $item_ids, $ship_date, $change, $change_id)
    public function scopeTemporaryShip($query)
```